src/Http/Controllers/NotificationsController.php

```diff
@@ -217,4 +227,49 @@ public function events(NotificationEventsRequest $request)
     {
         return $this->sendResponse($this->service->getEvents());
     }
+
+    /**
+     * @OA\Post(
+     *      path="/api/notifications/:notification/read",
```

Comment:
https://api-stage.escolalms.com/api/documentation

we use 

```suggestion
     *      path="/api/notifications/{notification}/read",
```

everywhere